signac/contrib/project.py

```diff
@@ -214,20 +219,23 @@ def workspace(self):
             such as $HOME."""
         return self._wd
 
+    @deprecated(deprecated_in="1.3", removed_in="2.0", current_version=__version__,
+                details="Use project.id instead.")
     def get_id(self):
         """Get the project identifier.
 
         :return: The project id.
         :rtype: str
-        :raises LookupError: If no project id could be determined.
         """
         try:
             return str(self.config['project'])
         except KeyError:
-            raise LookupError(
-                "Unable to determine project id ."
-                "Are you sure '{}' is a signac project path?".format(
-                    os.path.abspath(self.config.get('project_dir', os.getcwd()))))
+            return None
+
+    @property
+    def id(self):
+        """Get the project identifier."""
+        return self.get_id()
```

Comment:
@b-butler Flip the implementation so that the deprecated method calls the preferred property.